codegen/src/main/resources/templates/common/enum-class.ftl

```diff
@@ -49,4 +49,8 @@ public enum ${shape.shapeName} {
 
         throw new IllegalArgumentException("Cannot create enum from " + value + " value!");
     }
+
+    public static Set<${shape.shapeName}> knownValues() {
+        return Stream.of(values()).filter(v -> v != UNKNOWN_TO_SDK_VERSION).collect(toSet());
+    }
```

Comment:
Is this still used for anything?

Reply:
i think we can actually kill it - I'll check

Reply:
I'll do that as a different PR (there's a bunch that can go - pretty much all the model stuff)